### PR TITLE
Add nnunet scripts

### PR DESCRIPTION
**Pull request by hermancollin**
Solves #2

### CODE REVIEW

File: nnunet_scripts/convert_to_nnunetv2_format.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    #print("Renommage des images d'entraînement...")
+    #rename_images(os.path.join(output_folder, "imagesTr"), os.path.join(output_folder, "imagesTr"), '.png')
+
+    # Appeler la fonction de renommage pour les labels dans output_labels_folder
+    #print("Renommage des fichiers de labels...")
+    #rename_labels(os.path.join(output_folder, "labelsTr"))
+
+    # Créer le fichier dataset.json après la conversion
+    #create_dataset_json(output_folder, channel_names, labels, num_training, file_ending, overwrite_image_reader_writer)
+    #print("Conversion terminée avec succès !")
```
Comment (hermancollin) [inline nnunet_scripts/convert_to_nnunetv2_format.py]: we might not want the comments in the codebase

**Review by hermancollin — COMMENTED** [review]
Some general comments: 
- I'm glad you were able to make the preprocessing work! I would just like to trim some parts to clean the code a bit. Maybe we can do it together. 
- For now it will be fine but in the future you might want to write comments in english. I understand that if you are using an AI copilot of some sort and might prefer prompting it in french, but potential future readers of your code may not speak french!
- Overall, good job that's a nice first PR! The only issue is that the name of this PR is "Add reproducibility setup" so it's slightly out of scope. It's not a problem for now but in the future you might want to keep things more organized.

**Comment by carlosnazzarian** [comment]
> Some general comments:
> 
> * I'm glad you were able to make the preprocessing work! I would just like to trim some parts to clean the code a bit. Maybe we can do it together.
> * For now it will be fine but in the future you might want to write comments in english. I understand that if you are using an AI copilot of some sort and might prefer prompting it in french, but potential future readers of your code may not speak french!
> * Overall, good job that's a nice first PR! The only issue is that the name of this PR is "Add reproducibility setup" so it's slightly out of scope. It's not a problem for now but in the future you might want to keep things more organized.


Thank you for the feedback!  Let's definitely collaborate on streamlining the code further. I'll also start documenting in English to make it more accessible for everyone. Thanks for the heads up on the PR naming; I'll be more careful to match names to content in the future.

File: nnunet_scripts/convert_to_nnunetv2_format.py (hunk 1 above)
Comment (carlosnazzarian) [reply nnunet_scripts/convert_to_nnunetv2_format.py]: perfect, i will fix it

**Pull request merged**